src/parthenon_manager.cpp

```diff
@@ -352,6 +350,12 @@ void ParthenonManager::RestartPackages(Mesh &rm, RestartReader &resfile) {
           pmb->meshblock_data.Get()->GetVarPtr(label)->dealloc_count = dealloc_count;
         } else {
           // nothing to read for this block, advance reading index
+          IndexRange out_ib = v_info.cellbounds.GetBoundsI(theDomain);
+          IndexRange out_jb = v_info.cellbounds.GetBoundsJ(theDomain);
+          IndexRange out_kb = v_info.cellbounds.GetBoundsK(theDomain);
+          int nCells = (out_ib.e - out_ib.s + 1);
+          nCells *= (out_jb.e - out_jb.s + 1);
+          nCells *= (out_kb.e - out_kb.s + 1);
```

Comment:
This fixes a bug, where if fine fields were present the `index` jump was too large by a factor of $2^D$.

Reply:
This has since been fixed in another PR